File: connect_db.php
<?php
$serverName = "HEROSQL1";
$connectionOptions = array(
    "Database" => "Test",
    "Uid" => "vmuser",
    "PWD" => "[hero123]"
);

// Connect to SQL Server
$conn = sqlsrv_connect($serverName, $connectionOptions);

if ($conn === false) {
    die(print_r(sqlsrv_errors(), true));
}

// Example query
$sql = "SELECT * FROM users";
$stmt = sqlsrv_query($conn, $sql);

if ($stmt === false) {
    die(print_r(sqlsrv_errors(), true));
}

while ($row = sqlsrv_fetch_array($stmt, SQLSRV_FETCH_ASSOC)) {
    print_r($row);
}

sqlsrv_close($conn);
?>

File: connect_db.py
import pyodbc

# Database connection
conn = pyodbc.connect(
    'DRIVER={ODBC Driver 17 for SQL Server};'
    'SERVER=HEROSQL1;'
    'DATABASE=Test;'
    'UID=vmuser;'
    'PWD=[hero123]'
)

cursor = conn.cursor()

# Example query
cursor.execute('SELECT * FROM users')
rows = cursor.fetchall()

for row in rows:
    print(row)

conn.close()
